streampark-common/pom.xml

```diff
@@ -30,6 +30,20 @@
     <dependencies>
 
         <!-- test -->
+        <dependency>
```

Comment:
why add it？If it is used in the test use case, please go with the test use case, otherwise the developer will not be able to understand it.